src/Profiles.ts

```diff
@@ -305,6 +305,32 @@ export class Profiles { // Processing stops if there are no profiles detected
         return [updSession.ISession.user, updSession.ISession.password, updSession.ISession.base64EncodedAuth];
     }
 
+    private async rempromptUsername() {
```

Comment:
Does having two separate functions for re prompting make life easier because then the logic can be centralized and reused if even one of the two are wrong.

Reply:
I think it would be easier to re-prompt the whole profile.